activities/workshops/governance-exercise.md

```diff
@@ -0,0 +1,108 @@
+---
+type: resource
+---
+
+# Governance exercise
+
+This exercise lets the participants co-create a governance model. The participants will be handed a set of building blocks and instructions for two different scenarios. Each group will be facilitated during the process. Workshop time: 75-90 minutes.
+
+## The exercise have three phases
+
+Each of these should have an introduction to the task and explaining the expectations of the outcome so that everybody participating knows what to do.
+
+- A current situation
+
+  - Reason: to get their brains started and see that the process is both complex and probably incomplete.
+
+- An ideal situation
+
+  - Reason: If the participants get to design a new process after describing what they have, they will be part of a process of change. Since they themselves have identified a gap to where they want to be they have internalized that change is coming.
+
+- Discussion
+
+  - Reason: By discussing what they came up with they will realize what went through the different participants heads.
```

Comment:
This is invalid Markdown and might yield different results on different renders. 

I suggest using lists like:

```markdown
* A
  * b
* C
  * d
* E
  * f
```

however the following might be more fitting:

```markdown
1. A: b
2. C: d
3. E: f
```

or transforming it into headings with content.